src/App.css

```diff
@@ -6,24 +6,82 @@
 }
 
 .App {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  text-align: justify;
   font-size: 2.5rem;
```

Comment:
- [ ] [OPTIONAL] If you change the `font size` value to `1.5rem` it would good 👍🏼